lib/SlackBot.js

```diff
@@ -4,6 +4,7 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var querystring = require('querystring');
 var async = require('async');
+var Cryptr = require('cryptr');
```

Comment:
@abratnap I encrypt other data in the storage database being generated by botkit in a separate table, I use the slack_id as key.